.gitignore

```diff
@@ -168,4 +168,6 @@ tsdoc-metadata.json
 *.orig
 
 # cppwinrt
-nul
\ No newline at end of file
+nul
+
+out/
```

Comment:
should we make this an absolute path? Now every out anywhere in the repo is ignored...

Reply:
```suggestion
# WinMd2Md output run during PR and CI
/vnext/target/WinMd2MdOutput 
```
Note this could technically be revered for 2 reasons:
* vnext/target is alredy excluded
* this only runs in Pipelines and we don't push changes form pipeslines